src/shared/states/ChatStates.ts

```diff
@@ -0,0 +1,52 @@
+import axios from 'axios';
+import { create } from 'zustand'
+import ShortChatType from '../types/ShortChatType';
+
+interface ChatSlice {
+    loading: boolean,
+    success: boolean,
+    error: string,
+    chats: ShortChatType[],
+    currentChat: string,
+    filteredChats: ShortChatType[],
```

Comment:
I think you shouldn't store filtered chats here, it should be locally inside component. You don't need to use them somwhere else